core/conn/odb/src/odb.c

```diff
@@ -13133,7 +13134,7 @@ static int Otcol(int eid, SQLHDBC *Ocn)
                             str[j]= '\0';
                         }
                     } else {
-                        if ( isspace((int)str[j]) || str[j]=='#' )
```

Comment:
In Pg 53 of the ODB manual
( http://trafodion.incubator.apache.org/docs/odb/Trafodion_odb_User_Guide.pdf ) we have

"A generic mapfile contains:
• Comments (line starting with #)
• Mappings to link input file fields to the corresponding target table columns."

With this change can we still support comments in the mapfile?

In Pg 74, # is also used as the comment indicator for a file that lists the tables to be loaded/extracted. Does this still work?

~/Devel/odb $ cat tlist.txt
# List of tables to extract src=TRAFODION.MAURIZIO.ORDERS
src=TRAFODION.MAURIZIO.CUSTOMER src=TRAFODION.MAURIZIO.PART
src=TRAFODION.MAURIZIO.LINEITEM